FrontEnd/scripts/works.js

```diff
@@ -25,21 +26,34 @@ async function fetchWorkData() {
   return response;
 }
 
-async function displayWork() {
+const worksContainer = document.querySelector(".gallery");
```

Comment:
Maybe should we gather all querySelector declaration at the same level (beginning of the file is generally the best place)